cypress/e2e/file-upload-new-UI.cy.js

```diff
@@ -0,0 +1,33 @@
+describe("tests for uploaded files shown on editor", () => {
```

Comment:
Are these the changes from https://github.com/OpenCovenant/quill/pull/273?
Ideally, PRs should contain code that is not shared with other PRs. It would be awesome here if you could add a Cypress test regarding uploading a 2nd file.